spinalcordtoolbox/scripts/sct_apply_transfo.py

```diff
@@ -81,11 +81,13 @@ def get_parser():
         metavar=Metavar.file)
     optional.add_argument(
         "-x",
-        help=""" Interpolation method. The 'label' method is to be used if you would like to apply a transformation
-        on a file that has single-voxel labels (classical interpolation methods won't work, as resampled labels might
-        disappear or their values be altered). The function will dilate each label, apply the transformation using
-        nearest neighbour interpolation, and then take the center-of-mass of each "blob" and output a single voxel per
-        blob.""",
+        help="Interpolation method.\n"
+             "Note: The 'label' method is a special interpolation method designed for single-voxel labels (e.g. disc "
+             "labels used as registration landmarks). This method is necessary because classical interpolation may "
```

Comment:
```suggestion
             "labels used as registration landmarks, compression labels, etc.). This method is necessary because classical interpolation may "
```